service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedPlacementOrderEventHandlerTest.java

```diff
@@ -0,0 +1,228 @@
+            Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS, DEFAULT_ADMIN_EMAIL, CAFCASS_EMAIL_ADDRESS),
+            notifyData,
+            TEST_CASE_ID
+        );
+    }
+
+    @Test
+    void shouldSendOrderNotification_ToRepresentedParentsSolicitorByEmail_And_ToRepresentedChildSolicitorByEmail() {
+        Element<Respondent> anotherRespondent = testRespondent("Someone", "Else");
+        Element<Respondent> father = testRepresentedRespondentWithAddress("Father", "Jones", FATHER_SOLICITOR_EMAIL);
+        Element<Respondent> mother = testRepresentedRespondentWithAddress("Mother", "Jones", MOTHER_SOLICITOR_EMAIL);
+        Element<Child> child = element(Child.builder()
+            .solicitor(RespondentSolicitor.builder().email(PRIVATE_SOLICITOR_USER_EMAIL).build())
+            .build());
+        CaseData caseData = basicCaseData.toBuilder()
+            .respondents1(List.of(anotherRespondent, father, mother))
+            .children1(List.of(child))
+            .placementEventData(PlacementEventData.builder()
+                .placements(wrapElements(
+                    Placement.builder()
+                        .childId(child.getId())
+                        .noticeDocuments(wrapElements(
+                            PlacementNoticeDocument.builder()
+                                .respondentId(anotherRespondent.getId())
+                                .type(CAFCASS)
+                                .build(),
+                            PlacementNoticeDocument.builder()
+                                .respondentId(mother.getId())
+                                .type(PARENT_SECOND)
+                                .build(),
+                            PlacementNoticeDocument.builder()
+                                .respondentId(father.getId())
+                                .type(PARENT_FIRST)
+                                .build()
+                        )).build()
+                )).build()
+            ).build();
+        when(sealedOrderHistoryService.lastGeneratedOrder(any()))
+            .thenReturn(GeneratedOrder.builder().children(List.of(child)).build());
+        PlacementOrderIssuedNotifyData notifyData = mock(PlacementOrderIssuedNotifyData.class);
+        when(orderIssuedEmailContentProvider.getNotifyDataForPlacementOrder(caseData,
+            ORDER_NOTIFICATION_DOCUMENT,
+            child.getValue())).thenReturn(notifyData);
+
+        underTest.sendPlacementOrderNotification(new GeneratedPlacementOrderEvent(caseData,
+            ORDER_DOCUMENT,
+            ORDER_NOTIFICATION_DOCUMENT));
+
+        verify(notificationService).sendEmail(
+            PLACEMENT_ORDER_GENERATED_NOTIFICATION_TEMPLATE,
+            Set.of(FATHER_SOLICITOR_EMAIL, MOTHER_SOLICITOR_EMAIL, PRIVATE_SOLICITOR_USER_EMAIL),
+            notifyData,
+            TEST_CASE_ID
+        );
+    }
+
+    @Test
+    void shouldSendOrderNotification_ToUnrepresentedParentsByPost_And_NotToUnrepresentedChildSolicitorByEmail() {
```

Comment:
And_**Not**ToUnrepresentedChildSolicitorByEmail -> And_ToUnrepresentedChildSolicitorByEmail

Reply:
It's actually correct. Unrepresented children won't get a message.